tests/e2e/reconcile/01-install.yaml

```diff
@@ -15,6 +15,8 @@ kind: Microservices
 metadata:
   name: simplest
 spec:
+  images:
+    tempoQuery: quay.io/rvargasp/tempo-query
```

Comment:
What does this image contain?

Reply:
that image contains the jaegerQuery pluign that supports TLS communication with the tempo front-end. We can remove this when the new version of the oficial image is released.

Reply:
Could you add a comment explaining that? Even, I would create an issue in this repository to do that change in the future.